checker/src/main/java/org/checkerframework/checker/nullness/NullnessVisitor.java

```diff
@@ -271,7 +271,8 @@ public Void visitMemberSelect(MemberSelectTree tree, Void p) {
                 || tree.getExpression().getKind() == Tree.Kind.PARAMETERIZED_TYPE
                 // case 8. static member access
                 || ElementUtils.isStatic(e))) {
-            checkForNullability(tree.getExpression(), DEREFERENCE_OF_NULLABLE);
+            AnnotatedTypeMirror type = atypeFactory.getAnnotatedType(tree.getExpression());
+            checkForNullability(type, tree.getExpression(), DEREFERENCE_OF_NULLABLE);
```

Comment:
Shouldn't the second argument be `tree`? Otherwise, isn't this just inlining https://github.com/eisop/checker-framework/blob/a3bb7ac902a5b25b1caf159fc8151a7b0ce8156a/checker/src/main/java/org/checkerframework/checker/nullness/NullnessVisitor.java#L638 ?
Similar comment for the other change below.

Reply:
You're absolutely right. This is just inlining the fn call without making any changes to functionality.

Reply:
I'll make some fixes.